modules/floating-buttons/module.php

```diff
@@ -520,4 +522,20 @@ private function render_floating_buttons(): void {
 		}
 	}
 
+	/**
+	 * Register styles.
+	 *
+	 * At build time, Elementor compiles `/modules/floating-buttons/assets/scss/frontend.scss`
+	 * to `/assets/css/widget-floating-buttons.min.css`.
+	 *
+	 * @return void
+	 */
+	public function register_style() {
+		wp_register_style(
+			'widget-floating-buttons',
+			$this->get_css_assets_url( 'widget-floating-buttons', null, true, true ),
+			[],
+			ELEMENTOR_VERSION
+		);
+	}
```

Comment:
This is not needed with the Improved CSS loading. So I guess this pertains to my first question, did we now decide to improve the general CSS loading as well?

Reply:
Yes, this PR improves the general CSS loading, both for `frontend.css` and for widgets styles.

Reply:
@rami-elementor What is the point then of the CSS Loading Experiment if Elementor will now be so optimised out of the box? It seems like it's now just a choice between if you want to load it in the <head> or inside the widget 🤔

Reply:
I understand previously - having an experiment, to make a drastic improvement, but now it kind of just changes the location of the loading. So maybe the "loading location" can just be part of the widget config or something. I hope my thought pattern is making sense here :)